src/coffee/directives/api/markers.coffee

```diff
@@ -15,6 +15,7 @@ angular.module("uiGmapgoogle-maps.directives.api")
           type: '=?type' # cluster, spider, default undefined - normal
           typeOptions: '=?typeoptions'
           typeEvents: '=?typeevents'
+          deepComparison: '=?deepcomparison'
```

Comment:
Actually you will want to add this scope attribute [here](https://github.com/angular-ui/angular-google-maps/blob/master/src/coffee/directives/api/plural.coffee#L37) . That way this gets worked into all plural directives in one shot.